internal/dag/gatewayapi_processor.go

```diff
@@ -78,6 +79,10 @@ type GatewayAPIProcessor struct {
 
 	// GlobalCircuitBreakerDefaults defines global circuit breaker defaults.
 	GlobalCircuitBreakerDefaults *contour_api_v1alpha1.GlobalCircuitBreakerDefaults
+
+	// UpstreamTLS defines the TLS settings like min/max version
+	// and cipher suites for upstream connections.
+	UpstreamTLS *UpstreamTLS
```

Comment:
looks like this needs to be set in `cmd/serve.go` so general upstream TLS settings can flow through

we can add some follow up e2e tests for these parameters (tls version etc.)

Reply:
looks like it is set in `featuretests` but not `cmd/serve.go` (which unfortunately only gets covered by e2es)

Reply:
Good catch. I added the upstreamTLS to the processor in `cmd/serve.go`, and we can follow up with UpstreamTLS testing at the e2e level in a follow up PR if we want to do that as suggested.